subjects/countstars/README.md

```diff
@@ -0,0 +1,31 @@
+## CountStars
+
+### Instructions
+Write a function named `CountStar` that makes u fall asleep by counting stars, it takes an integer in the parameter
+- If the number is negative or equal to 0, return "`No star`"
+- No need to manage overflow.
+
+```go
+func CountStars(num int) string {
+
+}
+```
+```go
+import (
+	"fmt"
+	"strconv"   
+)
+
+func main() {
+	fmt.Println(CountStars(5))
+	fmt.Println(CountStars(4))
+	fmt.Println(CountStars(-1))
+}
+```
+
+```console
+$ go run . | cat -e
+1 star...2 star...3 star...4 star...5$
+1 star...2 star...3 star...4$
+No star$
```

Comment:
Shouldn't the last number also have the "start"? 

For example, CountStars(5) shouldn't be "1 star...2 star...3 star...4 star...5 star" instead of "1 star...2 star...3 star...4 star...5".

One improvement could be if the number of starts is bigger than 1, using starts instead of start. Using the same example of CountStars(5), it would be "1 star...2 stars...3 stars...4 stars...5 stars".

Also, the "$" should be removed as it suggests that the function should return it.